index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" type="image/*" href="assets/images/portfolio.png">
+    <link rel="stylesheet" href="./assets/style/main.css">
+    <title>First Mobile version Portfolio</title>
+</head>
+<body>
+        <header class="header-content">
+            <span><a href="#"  id="mylogo" class="common-font-family">My Logo</a></span> 
+            <nav id="nav-ham-icon"><span id="hamburgur-icon">&#9776;</span></nav>
+        </header>
+        <main class="wrap-content">
+                <h1 id="myfullname" class="common-font-family">I’m Osman Wako
+                    Glad to see you!</h1>
```

Comment:
- [ ] As mentioned by the previous reviewer, it is required to stick to the figma design as much as possible. This includes font-size, font-family, color, background-color, margin, padding among others. At this moment, some of these are not the same on your design and the figma design. Figma design uses `font-family: Poppins` for the content of `<h1>`. You specified it inside body selector but it cannot work now until you import it from [google font](https://fonts.google.com/). Because it is not available by default on your system. Kindly follow that link and implement  it accordingly

- [ ] You can use `<br />` to break the content of your `<h1>` so as to look exact like how it appears on figma design

```
  <h1>I'm Osma Wako <br /> Glad to see you</h1>
```